Add Security section to docs (#378)
0xPolygon/polygon-docs: docs/security/disclosure.md

> Email to: security@polygon.technology

Reviewer: ```suggestion
Email: security@polygon.technology
```

> Have you discovered a security flaw or CVE related to our technology? Please notify us before informing the outside world, so that we can first take action. Doing so is called ‘responsible disclosure’.

Reviewer: ```suggestion
Have you discovered a security flaw or CVE related to our technology? Please notify us before informing the outside world, so that we can take action. This is known as ‘responsible disclosure’.
```

> Email us at security@polygon.technology, and please make use of the below PGP key o encrypt any sensitive data. 
> 
> Looking for a reward or bounty related something you personally discovered, submit to one of our [Bug bounty programs](bugbounty.md).

Reviewer: ```suggestion
Email us at security@polygon.technology, and use the PGP key below to encrypt any sensitive data. 

If you are looking for a reward or bounty related something you personally discovered, submit your request to one of our [bug bounty programs](bugbounty.md).
```
